core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -442,7 +442,7 @@ public void heartbeat() {
         String msg = String.format(
             "ClockTime: %d, LastActivityTime: %d. Worker %d register stream hanging for %sms!"
                 + " Tune up %s if this is undesired.",
-            clockTime, lastActivityTime, context.mWorker.getId(), staleTime,
+            clockTime, lastActivityTime, context.getWorkerInfo().getId(), staleTime,
```

Comment:
There is a `context.getWorkerId()` btw. It's less verbose to directly use that. Or maybe remove that function and change all call sites to context.getWorkerInfo().getId()`.

Reply:
Sounds good, I will post another pr to remove the getWorkerId, thx!